client/src/components/Container/Container.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { View, TouchableWithoutFeedback, Keyboard } from 'react-native';
+import PropTypes from 'prop-types';
+import styles from './styles';
+
+const Container = ({ children, backgroundColor }) => {
+  const containerStyles = [styles.container];
+
+  if (backgroundColor) {
+    containerStyles.push({ backgroundColor });
+  }
+
+  return (
+    <TouchableWithoutFeedback onPress={() => Keyboard.dismiss()}>
```

Comment:
Touch anywhere on the screen to dismiss keyboard - I can remove and re-add later if we need it.

Reply:
I just assumed this wasn't necessary because tapping outside the field would drop focus but that appears to not be the case so this sounds good to me.